### PR TITLE
[WIP] started on envvars

### PR DESCRIPTION
**Pull request by rosscdh**
Very rough start still WIP

### CODE REVIEW

File: test/e2e/project_management_test.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ func TestProjectCreation(t *testing.T) {
 	projectName := "proj-" + fixture.Name()
 	_, err := fixture.RunCli("proj", "create", projectName,
 		"--description", "Test description",
+		"-e", "MY_PARAM=MY_PARAM_VALUE",
```
Comment (rosscdh) [inline test/e2e/project_management_test.go]: needs to be in apps not project

**Comment by alexec** [comment]
Hey @rosscdh - let me know how's it going and if you want any help

**Comment by rosscdh** [comment]
> Hey @rosscdh - let me know how's it going and if you want any help

Thanks alex, just got back in after some time off have to get the day job stuff going .. am a little lost as the next steps, but so far does it look ok?

**Comment by rosscdh** [comment]
Sorry doesnt look like im going to have time for this PR for some weeks :(

**Comment by alexec** [comment]
See https://github.com/argoproj/argo-cd/pull/1860

**Comment by alexec** [comment]
I've finished the backend implemenation for this. I've not added and CLI parameters to support this (you could use `app patch`).

**Comment by alexec** [comment]
@rosscdh we're having a clean up, and this hasn't moved for a while. I'm going to mark label as "stale".

**Comment by alwaysastudent** [comment]
@alexec  does custom plugins support parameter overrides from the command line?

**Pull request closed without merging**